cool-core/src/main/java/com/nus/cool/core/util/parser/CsvTupleParser.java

```diff
@@ -33,4 +33,9 @@ public String[] parse(Object tuple) {
     String record = (String) tuple;
     return record.split(",", -1);
   }
+
+  public static String[] Parse(Object tuple) {
```

Comment:
Why add this static function?

Reply:
No need to create a object.
Use static is enough, no other attribute is need.

Reply:
I mean that why don't you use the non-static version.

Reply:
fix